lib/docx/body-reader.js

```diff
@@ -89,17 +89,12 @@ function BodyReader(options) {
         var sectPrElement = element.first('w:sectPr');
         if ( sectPrElement ){
 
-            var colCount = sectPrElement
-                .first("w:cols")
-                .attributes["w:num"];
-
 			var rawFooterReferences = sectPrElement.getElementsByTagName("w:footerReference");
-			var footerReferences = []
+			var footerReferences;
```

Comment:
Set this as var `footerReferences = null` ?

Then line 98 can be refactored to `footerReferences: footerReferences`